### ISSUE TITLE
Not possible to set the name of the part of the ExtensionBasedTextEditor

### ISSUE THREAD

**Issue by rubenporras** [eclipse-platform/eclipse.platform.ui#881]
There is no extension point where one can insert code to set the name of the part, and org.eclipse.ui.part.WorkbenchPart.setPartName(String), so it is also not possible to set it later from outside.

Would it be possible to offer an extension point for this matter?

**Comment by mickaelistria** [comment]
Yes, I think it would be good. However, before starting a new extension point, I'm wondering whether there might be already existing extension points, or adapters, or APIs for that we could reuse.

**Comment by rubenporras** [comment]
I could not find any but I do not know this part of the platform very well. If you now some extension point or API I could try I am happy to do that.
